lib/lookbook/utils.rb

```diff
@@ -6,7 +6,7 @@ module Utils
     protected
 
     def generate_id(*args)
-      parts = args.map { |arg| arg.to_s.parameterize.underscore }
+      parts = args.map { |arg| arg.to_s.force_encoding("UTF-8").parameterize.underscore }
```

Comment:
Was getting `ASCII-8BIT` errors when embedding previews in the dummy app, not just on Tabs but on any Page... perhaps this was my own issue, but adding this fixed the issue.